node/src/index.ts

```diff
@@ -459,6 +459,84 @@ export class LocalTable<T = number[]> implements Table<T> {
   async delete (filter: string): Promise<void> {
     return tableDelete.call(this._tbl, filter).then((newTable: any) => { this._tbl = newTable })
   }
+
+  /**
+   * Clean up old versions of the table, freeing disk space.
+   *
+   * @param olderThan The minimum age in minutes of the versions to delete. If not
+   *                  provided, defaults to two weeks.
+   * @param deleteUnverified Because they may be part of an in-progress
+   *                  transaction, files newer than 7 days old are not deleted
+   *                  by default. If you are sure that there are no in-progress
+   *                  transactions, then you can set this to `true` to delete all
+   *                  files older than `older_than`.
```

Comment:
Do you think this detail is something that a lancedb user is going to want to internalize.  We should still delete most files if this is false anyways.  The only things that really get missed are leftover files from aborted transactions which should (hopefully) be quite rare.

Reply:
I think providing some way to cleanup aborted transactions is important. Basically the TLDR for this options is "If I choose True here, I promise there aren't any ongoing writes happening".